internal/pkg/cli/pipeline_update.go

```diff
@@ -278,7 +300,17 @@ func buildPipelineUpdateCmd() *cobra.Command {
 			if err := opts.Validate(); err != nil {
 				return err
 			}
-			return opts.Execute()
+			if err := opts.Execute(); err != nil {
+				return err
+			}
+			if opts.shouldPromptUpdateConnection {
```

Comment:
Maybe we can delete this code since it will be replaced. What do you think?

Reply:
How about we delete it when we replace it? Does that work?